deltachat-ios/Controller/ProfileInfoViewController.swift

```diff
@@ -131,6 +131,10 @@ class ProfileInfoViewController: UITableViewController {
     @objc private func doneButtonPressed(_ sender: UIBarButtonItem) {
         dcContext.displayname = displayName
         self.dismiss(animated: true, completion: nil)
+
+        if let appDelegate = UIApplication.shared.delegate as? AppDelegate {
```

Comment:
Yep, I would not pull in appDelegate.appCoordinator here. 
I'll provide fix for this.